### DISCUSSION TITLE
Ideas💡family dynamic/ Giving birth/ pets

### DISCUSSION THREAD

**Discussion by Ajohnson204** [QI-Software/deadmatterpublic#214] [Ideas]
I think there should be a family situation we're u can assign different names of family members/friends

I also think there should be a system where birth is a thing in females too make it more realistic in a post apocalyptic world.

And also being able to have a system where u can conceive and raise a child or a child that was left behind in the apocalypse as well!!

I also think there should be pets you can raise/from puppy's or bigger dogs that are now stray dogs and they can help fight off zombies and enemies

**Comment by johnnyguitarFP** [comment]
I don't think some of these suggestions are appropriate for the game we're building.
